common/src/main/resources/assets/voicechat/lang/en_us.json

```diff
@@ -1,217 +1,212 @@
+  "message.voicechat.mute.disabled_ptt": "You can't mute your microphone when using Push to Talk.",
+  "message.voicechat.disable.enabled": "Press to enable all voice chat sounds.",
+  "message.voicechat.disable.disabled": "Press to disable all voice chat sounds.",
+  "message.voicechat.disable.no_speaker": "No speakers available.",
+  "message.voicechat.hide_icons.enabled": "Press to show all voice chat icons.",
+  "message.voicechat.hide_icons.disabled": "Press to hide all voice chat icons.",
+  "message.voicechat.show_group_hud.enabled": "Press to hide the group chat HUD.",
+  "message.voicechat.show_group_hud.disabled": "Press to show the group chat HUD.",
+  "message.voicechat.recording.disabled": "Press to start recording.",
+  "message.voicechat.recording.enabled": "Press to stop recording.",
   "message.voicechat.no_speak_permission": "You do not have permission to speak",
   "message.voicechat.no_listen_permission": "You do not have permission to hear the voice chat",
   "message.voicechat.no_group_permission": "You do not have permission to join groups",
   "message.voicechat.search_hint": "Search...",
   "message.voicechat.search_empty": "Couldn't find any players with that name",
   "message.voicechat.other_volume": "Other",
-  "message.voicechat.other_volume.description": "The volume of any sound not listed below",
+  "message.voicechat.other_volume.description": "The volume of any sound not listed below.",
   "message.voicechat.more_members": "and %s more...",
   "message.voicechat.no_groups": "No existing groups",
-  "message.voicechat.audio_type": "3D audio: %s",
+  "message.voicechat.audio_type": "3D Audio: %s",
   "message.voicechat.audio_type.normal": "Normal",
   "message.voicechat.audio_type.reduced": "Reduced",
-  "message.voicechat.audio_type.off": "Off",
-  "message.voicechat.invite_player": "Invite %s to your group",
-  "message.voicechat.press_to_reassign_key": "Press to reassign key",
+  "message.voicechat.audio_type.off": "OFF",
+  "message.voicechat.invite_player": "Press to invite %s to your group.",
+  "message.voicechat.press_to_reassign_key": "Press to assign a key",
   "message.voicechat.set_up": "Press %s to set up",
   "message.voicechat.onboarding.reset": "Onboarding status has been reset",
   "message.voicechat.onboarding.next": "Next",
   "message.voicechat.onboarding.confirm": "Confirm",
   "message.voicechat.onboarding.back": "Back",
   "message.voicechat.onboarding.cancel": "Cancel",
   "message.voicechat.onboarding.introduction.title": "Set Up %s",
-  "message.voicechat.onboarding.introduction.description": "This setup guide helps you to configure your voice chat.\n\nIf you know what you are doing, you can skip this.",
-  "message.voicechat.onboarding.introduction.skip": "I know what I'm doing - Skip",
+  "message.voicechat.onboarding.introduction.description": "This setup guide will help you configure your voice chat.\n\nIf you know what to do, you can skip it.",
```

Comment:
I did this intentionally.

In my opinion, the sentence "If you know what to do, you can skip it" clearly and with no hidden message suggests that if you already know what you should do to set up the voice chat for yourself (because, for example, you are a longtime user or you got instructions from someone), you can safely skip this screen since you already know everything you need to know and the setup guide won't provide any new information.

To me, the sentence "If you know what you are doing, you can skip it" suggests with a subtle message that if you are confident, believe in yourself, fully understand what you are doing, and it wasn't just a misclick, then you can skip this screen because you have enough confidence in your skills.

In conclusion, I think the first sentence is definitely better than the second one in this context.

Reply:
In case someone used the mod before the onboarding was introduced, all settings would already be correct. If a player knows this they can skip it. "What to do" would imply theres something you need to do after skipping.

Reply:
"What to do" in this context can also refer to the user's knowledge of how to use this mod. If they already know what to do to make others hear them the way they want, then they can skip the setup guide.

Do you think this is the better of the two options?:
![Setup-guide-initial-screen-message](https://github.com/user-attachments/assets/eb460447-f941-481b-ad17-02bcf183ceb8)

Reply:
This is basically changing `this` to `it`?

Reply:
Yes and no. My intention was to show you a screenshot of the initial screen of the setup guide so that you could see the phrase "what you are doing" in context (which you seem to prefer?), as opposed to "what to do" (which is what I suggest in this PR).

Reply:
I'd like to keep the "what you are doing" personally.

Reply:
One last question from me on this topic then. What would you say about the following modification of the second sentence, as shown in the screenshot below? Better or worse?

![Setup-guide-initial-screen-alternative-message](https://github.com/user-attachments/assets/55a2ce2a-addf-4611-9215-2ba104f886a2)

Reply:
Thats fine.